training/DeepSpeed-Domino/requirements.txt

```
@@ -0,0 +1,6 @@
+apex
+deepspeed
```

Remember to update this like below to match the deepspeed release. 
```suggestion
deepspeed>=version
```

---

Sure, the release version is still uncertain. I'll provide an update once domino is merged into deepspeed.